autoextract_poet/page_inputs.py

```diff
@@ -0,0 +1,59 @@
+from typing import ClassVar, List, Optional, Type
+
+import attr
+
+from autoextract_poet.items import (
+    Article,
+    Item,
+    Product,
+)
+
+
+@attr.s(auto_attribs=True)
+class ResponseData:
```

Comment:
What do you think about renaming to to _AutoextractResponseData, to 1) avoid confusion with web-poet ResponseData, and 2) make it clear this is internal implementation detail, not a part of Python API users should work with?

Reply:
If we are going to remove `Response` from the other classes, we should remove also here. e.g. `_AutoextractData`

Reply:
I like this proposal. I'm renaming it to `_ResponseData` because it addresses the private class name and it's also a shorter variation of the name (we can already infer it's an AutoExtract response data from the package name and from the class' docstring).

Reply:
I agree with @ivanprado, therefore the class name was renamed to `_AutoExtractData`.